### DISCUSSION TITLE
Switch to specify host separately from the path

### DISCUSSION THREAD

**Discussion by jakub-id** [curl/curl#10108] [Ideas]
I use curl for testing services with the same paths but on different hosts. It would be convenient to specify the host separately from the path with something like `--host` and have curl accept just the path. It would work especially nice with curlrc. Or maybe there is already similar functionality that I don't know about?

**Comment by bagder** [comment]
curl already offers `--resolve` and `--connect-to` for this. Described somewhat in my old blogpost [curl another host](https://daniel.haxx.se/blog/2018/04/05/curl-another-host/).
